src/resources/Connectivity/Connectivity.ts

```diff
@@ -1,12 +1,13 @@
 import Resource from '../Resource';
 import API from '../../APICore';
-import {LogRequest, LogRequestId, LogRequestResult} from './ConnectivityInterface';
+import {LogRequest, LogRequestId, LogRequestResourceType, LogRequestResult} from './ConnectivityInterface';
 
 export default class Connectivity extends Resource {
     static baseUrl = `/rest/organizations/${API.orgPlaceholder}/logrequests`;
 
-    requestLog(body: LogRequest) {
-        return this.api.post<LogRequestId>(Connectivity.baseUrl, body);
+    requestLog(body: LogRequest, type?: LogRequestResourceType) {
+        const url = type ? this.buildPath(Connectivity.baseUrl, type) : Connectivity.baseUrl;
+        return this.api.post<LogRequestId>(url, body);
```

Comment:
nit-pick:
If there's a default value in the backend for `type`, could be nice to have it in the function directly.

There's no change in the behaviour, true, but at least the function signature is more representative of what's actually going on.

If the 'not-set' has a specific behaviour, nvm me ;)